homeassistant/components/brunt/cover.py

```diff
@@ -100,17 +100,17 @@ def __init__(
 
         self._remove_update_listener = None
 
-        self._attr_name = self._thing.NAME
-        self._attr_device_class = CoverDeviceClass.SHADE
+        self._attr_name = self._thing.name
+        self._attr_device_class = CoverDeviceClass.BLIND
```

Comment:
I suggest we change the device class in a separate PR since it's not required for the library bump, correct?